### DISCUSSION TITLE
Preview command from FZF_DEFAULT_OPTS

### DISCUSSION THREAD

**Discussion by timsofteng** [Aloxaf/fzf-tab#312] [General]
Hello. Is it possible to use same preview command in `fzf-tab` as I configured in `FZF_DEFAULT_OPTS`?
I use this command:
`export FZF_DEFAULT_OPTS="--preview 'bat {}' --preview-window hidden:wrap -m --bind=ctrl-o:toggle-preview,ctrl-i:toggle-down,ctrl-a:toggle-all"`

**Comment by Aloxaf** [comment]
fzf-tab won't parse `FZF_DEFAULT_OPTS`, you have to this manually.
```
zstyle ':fzf-tab:complete:*' fzf-preview 'bat $realpath'
zstyle ':fzf-tab:complete:*' fzf-flags --preview-window hidden:wrap
```

**Comment by timsofteng** [comment]
Cool!
Thanks.
